hera_cal/io.py

```diff
@@ -243,15 +247,131 @@ def load_cal(input_cal, return_meta=False):
         return gains, flags
 
 
-def write_cal():
-    '''TODO: copy over code from hera_cal.cal_formats.HERACal'''
-    raise NotImplementedError('This function has not been implemented yet.')
+def write_cal(fname, gains, freqs, times, pols, flags=None, quality=None, write_file=True,
+              return_uvc=True, outdir='./', overwrite=False, gain_convention='divide', 
+              history=' ', x_orientation="east", telescope_name='HERA', cal_style='redundant',
+              **kwargs):
+    '''Format gain solution dictionary into pyuvdata.UVCal and write to file
+
+    Arguments:
+        fname : type=str, output file basename
+        gains : type=dictionary, holds complex gain solutions. keys are antenna + pol
+                tuple pairs, e.g. (2, 'x'), and keys are 2D complex ndarrays with time
+                along [0] axis and freq along [1] axis.
+        freqs : type=ndarray, holds unique frequencies channels in Hz
+        times : type=ndarray, holds unique times of integration centers in Julian Date
+        pols : type=ndarray, holds unique polarization of gains in 'x' and/or 'y' form
```

Comment:
"...gains in string form. Can be either 'x' or 'y'".

Reply:
got rid of pol argument actually, can get it from the gains dictionary